### PR TITLE
Fixed desktop plugin implementation

### PR DESCRIPTION
**Pull request by DenchikBY**

### CODE REVIEW

File: lib/src/file_picker_io.dart [hunk 1]
```diff
@@ -8,8 +8,13 @@ import 'package:flutter/services.dart';
 
 import 'file_picker_result.dart';
 
-const MethodChannel _channel =
-    MethodChannel('miguelruivo.flutter.plugins.filepicker');
+final MethodChannel _channel = MethodChannel(
```
Comment (miguelpruivo) [inline lib/src/file_picker_io.dart:11]: Why can’t it be `const`?

**Review by miguelpruivo — CHANGES_REQUESTED** [review]
Thank you for the PR. Please, check the comments.

File: lib/src/file_picker_io.dart (hunk 1 above)
Comment (DenchikBY) [reply lib/src/file_picker_io.dart:11]: Because there is ternaty

Comment (miguelpruivo) [reply lib/src/file_picker_io.dart:11]: Oh yes, never mind. I’m reviewing on my phone. 😄

**Comment by kuhnroyal** [comment]
As far as I can tell, this broke mocking of the method channel. It is no longer const and it is private which means I can no longer access the same const instance just with the name to set the mock method handler.

**Comment by kuhnroyal** [comment]
Probably better to create 2 const channels and switch them depending on the platform.

**Pull request merged**